projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -109,7 +109,10 @@ export const Publicize = withModuleSettingsFormHelpers(
 						<ConnectUserBar
 							feature="publicize"
 							featureLabel={ __( 'Publicize', 'jetpack' ) }
-							text={ __( 'Sign in to connect your social media accounts.', 'jetpack' ) }
+							text={ __(
+								'Connect your WordPress.com account to connect your social media accounts.',
```

Comment:
This is the only one that raises some concerns to me — would suggested either of these with a slight preference for the first:

- Connect your WordPress.com account to manage your social media accounts.
- Connect your WordPress.com account to add your social media accounts.

Reply:
Done in 86f848d